litmus/preSi.ml

```diff
@@ -1355,12 +1355,14 @@ module Make
           O.o "}" ;
           O.o "" ;
           O.o "static void vars_free(vars_t *_vars) {" ;
-          let open OutUtils in
-          List.iter
-            (fun (n,_) ->
-              O.fi "free_pages(_vars->%s);" (fmt_code n))
-            test.T.code ;
-          O.o "}" ;        
+          if do_self then begin
```

Comment:
Oups, thanks

Reply:
How can I help get this in? Should I create a separate PR? Or do you want to cherry-pick it by hand?

Reply:
A separate PR would be the best method I guess. If you do it, I'll merge it immediately.